### ISSUE TITLE
toggles to single methods

### ISSUE THREAD

**Issue by ichdertobi** [alexa-d/alexa-openwebif#120]
It would be better to rebuild the toggles on individual methods for every action.
Thus, for example, with a command "sound off", the sound can not be switched on.
The OpenWebif API makes it possible to query the current state (muted, in standby, ..).
The single method could get the current status first and then it could react on this state and not simply toggle the state.

**Comment by fabsi88** [comment]
Yes, you are right. We also had thoughts about this. The problem is the timeout of 3 seconds... We have cases where 1 call is not fast enough ... So we don't know if this would work with 2 calls. But we will have a look into it and will test it if theres time

**Comment by ichdertobi** [comment]
Hm if we get timeout-problems, we only could try to create a new API-method for OpenWebif.
It is Opensource so it should be possible. I checked the Enigma 2 code and the OpenWebif Code but both doesnt support, what we need. But it shouldnt be very hard to create the new method to the API. But I dont know how fast they will merge it to theire master.

**Comment by fabsi88** [comment]
Jeah...thats something I thought as well for other (not yet implemented) methods. But I raised my white flag very fast 🏳️ It didn't looked intuitive for me...but hey..if you have time for this - have a look into!
